sdk/src/azure/platform/az_curl.c

```diff
@@ -305,8 +305,7 @@ static AZ_NODISCARD az_result _az_http_client_curl_send_get_request(CURL* ref_cu
 /**
  * handles DELETE request
  */
-static AZ_NODISCARD az_result
-_az_http_client_curl_send_delete_request(CURL* ref_curl)
```

Comment:
Once this PR is on top of master, hopefully we won't see this in the diff anymore :)